src/elements/tag/tag/tag.ts

```diff
@@ -41,9 +45,17 @@ class SbbTagElement extends SbbIconNameMixin(SbbDisabledTabIndexActionMixin(SbbB
   public accessor amount: string = '';
 
   /** Whether the tag is checked. */
-  @forceType()
-  @property({ reflect: true, type: Boolean })
-  public accessor checked: boolean = false;
+  @property({ reflect: false, type: Boolean })
+  public set checked(value: boolean) {
+    this._checked = value;
```

Comment:
maybe setting ariaPressed should come here as well? or updateformValue goes into willupdate?